### PR TITLE
update client.py with constants

### PR DESCRIPTION
**Pull request by swathipil**
updated for receive event scenario: https://github.com/Azure/azure-sdk-for-python/pull/19748/files

### CODE REVIEW

File: uamqp/client.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 from .receiver import ReceiverLink
 from .sasl import SASLTransport
 from .endpoints import Source, Target
+from .error import AMQPConnectionError
```
Comment (yunhaoling) [inline uamqp/client.py:25]: is this a duplicate of line 39?

File: uamqp/client.py [hunk 2]
```diff
@@ -291,14 +294,19 @@ def mgmt_request(self, message, operation=None, operation_type=None, node='$mana
...
         status, response, description = mgmt_link.execute(
             message,
             operation=operation,
             operation_type=operation_type,
             timeout=timeout
         )
+        status_code_field = kwargs.pop('status_code_field', b'statusCode')
+        description_field = kwargs.pop('description_field', b'statusDescription')
+        response.application_properties[status_code_field] = status
+        response.application_properties[description_field] = description
```
Comment (yunhaoling) [inline uamqp/client.py:309]: we could remove those first..

I think those field should be handled during mgmt link initialization:
https://github.com/Azure/azure-uamqp-python/blob/pyproto/uamqp/management_link.py#L52-L53

let's revisit this in the work for supporting mgmt link

File: uamqp/client.py [hunk 3]
```diff
@@ -555,9 +562,6 @@ def _message_received(self, message):
             self._message_received_callback(message)
         if not self._streaming_receive:
             self._received_messages.put(message)
-        elif not message.settled:
```
Comment (yunhaoling) [inline uamqp/client.py:558]: how about comment out those lines, and add a TODO: do we need settled property for a message?

File: uamqp/client.py (hunk 1 above)
Comment (swathipil) [reply uamqp/client.py:25]: ah yes it is, good catch

**Review by yunhaoling — APPROVED** [review]
LGTM!

**Pull request merged**